Workbooks/UsageMetrics/Usage - Users/Usage - Users.workbook

```diff
@@ -0,0 +1,710 @@
+    {
+      "type": 9,
+      "content": {
+        "version": "KqlParameterItem/1.0",
+        "parameters": [
+          {
+            "id": "3393f364-da6d-45f7-8457-20ab3b738318",
+            "version": "KqlParameterItem/1.0",
+            "name": "Cohort",
+            "type": 1,
+            "value": "",
+            "isHiddenWhenLocked": true
+          }
+        ],
+        "style": "above",
+        "queryType": 0,
+        "resourceType": "microsoft.insights/components"
+      },
+      "name": "parameters - 15"
+    },
+    {
+      "type": 9,
+      "content": {
+        "version": "KqlParameterItem/1.0",
+        "parameters": [
+          {
+            "id": "ff00e059-7ba9-4dfa-a926-8288cf30fc05",
+            "version": "KqlParameterItem/1.0",
+            "name": "UsersList",
+            "label": "Show",
+            "type": 2,
+            "isRequired": true,
+            "query": "datatable(value:string, text:string)[\r\n'user_Id', 'All Users', \r\n'user_AuthenticatedId', 'Authenticated Users'\r\n]",
+            "value": "user_Id",
+            "typeSettings": {
+              "additionalResourceOptions": []
+            },
+            "queryType": 0,
+            "resourceType": "microsoft.insights/components"
+          },
+          {
+            "id": "0afb0089-98e9-479c-b12e-a982870ce6be",
+            "version": "KqlParameterItem/1.0",
+            "name": "Whoused",
+            "label": "Who used",
+            "type": 2,
+            "isRequired": true,
+            "query": "datatable(value:string, text:string)[\r\n'pageViews,customEvents,requests', 'Any Custom Events, Request or Page View', \r\n'pageViews,customEvents', 'Any Custom Events or Page View', \r\n'customEvents','Any Custom Event', \r\n'pageViews','Any Page View', \r\n'requests','Any Request'\r\n]",
+            "value": "pageViews,customEvents",
+            "typeSettings": {
+              "additionalResourceOptions": []
+            },
+            "queryType": 0,
+            "resourceType": "microsoft.insights/components"
+          },
+          {
+            "id": "37b05d6c-ab7c-4190-94d8-4973fcf06c49",
+            "version": "KqlParameterItem/1.0",
+            "name": "TimeRange",
+            "label": "During",
+            "type": 2,
+            "isRequired": true,
+            "query": "datatable(value:string, text:string)[\r\n'1h', 'Last hour', \r\n'24h', 'Last 24 hours', \r\n'48h', 'Last 48 hours', \r\n'3d', 'Last 3 days',\r\n'7d', 'Last 7 days',\r\n'14d', 'Last 14 days',\r\n'30d', 'Last 30 days',\r\n'60d', 'Last 60 days',\r\n'90d', 'Last 90 days'\r\n]",
+            "value": "24h",
+            "typeSettings": {
+              "additionalResourceOptions": []
+            },
+            "queryType": 0,
+            "resourceType": "microsoft.insights/components"
+          },
+          {
+            "id": "3774b890-2a45-46d8-9a92-01145cc1c4ba",
+            "version": "KqlParameterItem/1.0",
+            "name": "WithGrain",
```

Comment:
you might want to try converting params like this that just do a "case" to use the new "criteria" parameter, that way you don't run queries

https://github.com/microsoft/Application-Insights-Workbooks/blob/master/Documentation/Parameters/Criteria.md

Reply:
I removed it because TimeRange changes to standard parameter type that includes a default grain.